privatelink/aws/dns-endpoints.sh

```diff
@@ -13,20 +13,24 @@
 #     *.usw2-az1                CNAME vpce-0123456789abcdef0-01234567-us-west-2a.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
 #     *.usw2-az3                CNAME vpce-0123456789abcdef0-01234567-us-west-2c.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
 #
+#   % ./dns-endpoints.sh vpce-0123456789abcdef0 usw2-az3
+#     *.usw2-az3                CNAME vpce-0123456789abcdef0-01234567-us-west-2c.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
```

Comment:
```suggestion
#   % ./dns-endpoints.sh vpce-0123456789abcdef0 usw2-az3
#     *                CNAME vpce-0123456789abcdef0-01234567-us-west-2c.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
```

The simplification is that the entire zone points to a single AZ.  The single zonal DNS will not be enough to function.  Bootstrapping happens on the `*` name and broker paths are on the zonal names (ie: `*.usw2-az1`).  With this example, bootstrapping is broken but direct to brokers in usw2-az3 work.